client/js/render.js

```diff
@@ -28,22 +28,11 @@ function buildChannelMessages(chanId, chanType, messages) {
 }
 
 function appendMessage(container, chanId, chanType, msg) {
+	let lastChild = container.find(".msg").last();
 	const renderedMessage = buildChatMessage(chanId, msg);
 
 	// Check if date changed
-	let lastChild = container.find(".msg").last();
 	const msgTime = new Date(msg.time);
-
-	// It's the first message in a window,
```

Comment:
Ah yes, this would be an issue after removing the logic for not creating a condense if there is only 1 message, as it woudl never be created until we had a normal message.